Zeplin.sketchplugin/Contents/Sketch/utils.cocoascript

```diff
@@ -1,8 +1,36 @@
+var isSelectionExportable = function (artboards) {
+    var artboardPixelLimit = 1000000000; // 1e9
+    var overSizedArtboardName = nil;
+
+    var artboardsLoop = [artboards objectEnumerator];
+    var artboard = nil;
+    while (artboard = [artboardsLoop nextObject]) {
+        var artboardSize = artboard.rect().size;
+        if (artboardSize.width * artboardSize.height > artboardPixelLimit) {
+            overSizedArtboardName = artboard.name();
```

Comment:
Would it be better to collect names of all over sized artboards and display warning for all at once instead of doing it just for the first one?

Reply:
Actually, we discuss this with @berkcebi and decided that displaying the first one will be enough since this is not a case frequently faced.

Reply:
We can reconsider this behavior, of course.